### DISCUSSION TITLE
Initial run dev not compiling files, wordpress critical error when theme activated

### DISCUSSION THREAD

**Discussion by stacymoorhead-njsba** [wprig/wprig#787] [Q&A]
Trying to run the initial npm run dev after setting everything up, but it doesn't seem to be compiling anything, and when I select the theme to activate it I get a wordpress critical error. I'm following the LinkedIn Learning tutorial and initially I was getting errors with require() needing to be "import from" in some of the dependencies, and was able to google enough to get past those issues and have the run dev process without error. Google basically told me to ignore the audit warnings and don't run a force audit fix. (I think the initial error is because I'm on the brand new Macbook Pro M1 with latest OS and there was issues with the versions of some of the node package versions). I spent a couple of days banging my head against a wall and I'm just at a loss for why it's not working. Not sure what to do next. Any insight would be much appreciated.

**Comment by roljohntorralba** [comment]
Hi, what versions of PHP, composer, node, and npm you're on?

I've recently switched to an M1 Pro system with the latest OS but haven't encountered any errors after running `npm run rig-init` for the first time.

I'm currently on these versions:
macOS 12.3
PHP 7.4.28
Composer 2.2.6
node v17.4.0
npm 8.3.1

**Reply by stacymoorhead-njsba** [reply]
I ended up abandoning ship on wprig and went back to underscores since it was a time sensitive project. I'll likely try the linkedIn learning tutorial again when I have some down time. 

These are my current versions:
macOS 12.2
PHP 8.1.2
Composer 2.2.5
node v16.13.2
npm 8.1.2

**Reply by dthenley** [reply]
Sorry you couldn't figure it out sooner. If you want some help a little sooner we also have a slack channel with a couple mac users.
https://join.slack.com/t/wprig/shared_invite/zt-162vhist3-XmeuQPLarTqw3~GrIlwIkA
